serializers.py

```diff
@@ -1573,6 +1570,23 @@ def post_process(self, review_assignment: ReviewAssignment, data: dict):
                     file = File.objects.get(pk=file_id)
                     review_assignment.review_round.review_files.add(file)
 
+        default_form = ReviewForm.objects.get(journal=review_assignment.article.journal, name="Default Form")
```

Comment:
Please add comments indicating the use case this part covers.

Reply:
I agree, a comment here would be helpful